guillotina/events.py

```diff
@@ -72,6 +73,11 @@ class ObjectMovedEvent(ObjectLocationEvent):
     """An object has been moved"""
 
 
+@implementer(IBeforeRenderView)
+class BeforeRenderView(ObjectEvent):
```

Comment:
Can we rename to `BeforeRenderViewEvent`.

Do not inherit from `ObjectEvent`, it's not an event on an object!

Need to implement `def __init__(self, request)` method

Reply:
oh ok!

Reply:
can I pass the view object aswell? `def __init__(self, request, view)`
I'd like to use it to get the view's route name, but not sure if that already comes in the request

Reply:
yup!